12/umbraco-cms/content-delivery-api.md

```diff
@@ -0,0 +1,458 @@
+        "linkedItem": {
+            "name": "Demo blog",
+            "route": {
+                "path": "/demo-blog/",
+                "startItem": {
+                    "id": "5d5ae914-9885-4ee0-a14b-0ab57f501a55",
+                    "path": "demo-blog"
+                }
+            },
+            "id": "5d5ae914-9885-4ee0-a14b-0ab57f501a55",
+            "contentType": "blog",
+            "properties": {
+                "title": "My demo blog",
+                "description": "Nihil incididunt dolores adipisicing placeat quisque imperdiet interdum autem, dolorem fusce rhoncus sunt leo inventore dictumst quisque, voluptatem, magni justo nostrud deserunt! Natus ipsam commodi dignissimos, sodales ab.\n"
+            }
+        }
+    },
+    "cultures": {}
+}
+```
+{% endcode %}
+
+
+
+The built-in property editors in Umbraco that allow for output expansion are:  &#x20;
+
+* `Umbraco.ContentPicker`
+* `Umbraco.MultiNodeTreePicker`
+
+</details>
+
+<details>
+
+<summary>Preview</summary>
+
+Similar to the preview concept in Umbraco, the Delivery API allows for requesting unpublished content through its endpoints. This can be done by setting a `Preview` header to `true` in the API request. However, accessing draft versions of your content nodes requires authorization via an API key configured in `appsettings.json` file - `Umbraco:CMS:DeliveryApi:ApiKey` setting. To obtain preview data, you must add the `Api-Key` request header containing the configured API key to the appropriate endpoints. Draft content is not going to be included in the JSON response otherwise.
+
+</details>
+
+<details>
+
+<summary>Localization</summary>
+
+If your content is available in multiple languages, the Delivery API can resolve localized content. When querying content by `id`, the `Accept-Language` header can be used to request variant content. When querying content by path, the culture is already known and included in the path, making the `Accept-Language` header unnecessary. However, if the header is present in the request, its value will take precedence over any other configuration settings. Localization is also supported by the means of the CMS's culture and hostname configuration.
+
```

Comment:
Would be nice with a request sample here that includes the `Accept-Language` header. Something like the request sample in the start item section.